include/bluetooth/hci.h

```diff
@@ -39,9 +39,15 @@ struct bt_hci_evt_hdr {
 #define BT_ACL_START_NO_FLUSH           0x00
 #define BT_ACL_CONT                     0x01
 #define BT_ACL_START                    0x02
+#define BT_ACL_COMPLETE                 0x03
 
-#define bt_acl_handle(h)                ((h) & 0x0fff)
+#define BT_ACL_POINT_TO_POINT           0x00
```

Comment:
Since these defines are used as bitmask, represent it as bit indices?
Same for the use of 12

Reply:
@cvinayak No they are not bitmasks. So using them as bitmask was wrong, I have reverted that change.